src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -216,6 +216,18 @@ export default class AircraftCommander {
     }
 
     /**
+     * Aborts an action. Deprecated.
+     *
+     * @for AircraftCommander
+     * @method runAbort
+     * @param {AircraftModel} aircraft
+     * @return {array} [success of operation, readback]
+     */
+    runAbort(aircraft) {
+        return [false, 'this command has been deprecated, clear an aircraft to another fix or heading instead'];
```

Comment:
let's change to `the 'abort' command has been deprecated, please see documentation for help`